test/ess_tests.jl

```diff
@@ -0,0 +1,88 @@
+using MCMCChains
```

Comment:
Maybe that would be a good time to structure the tests in a similar way as in other Turing packages by having one test file of the same name for each source file (i.e., `test/ess.jl` in this case). We should structure the source files better (in a separate PR), then we could do the same also for other tests.